src/BloomExe/Edit/EditingView.cs

```diff
@@ -763,6 +763,9 @@ void SaveChangedImage(GeckoHtmlElement imageElement, PalasoImage imageInfo, stri
 			{
 				if (ShouldBailOutBecauseUserAgreedNotToUseJpeg(imageInfo))
 					return;
+				// See https://silbloom.myjetbrains.com/youtrack/issue/BL-2776.
+				if (IsSameFile(imageElement, imageInfo))
```

Comment:
So, I'm not clear how this works. I open ImageToolbox, crop the image, will this preserve the cropping I did, or lose it?